vignettes/simulist.Rmd

```diff
@@ -0,0 +1,208 @@
+---
+title: "Getting Started with {simulist}"
+output: 
+  bookdown::html_vignette2:
+    code_folding: show
+pkgdown:
+  as_is: true
+vignette: >
+  %\VignetteIndexEntry{Getting Started with {simulist}}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This is an introductory vignette to the {simulist} R package. {simulist} simulates epidemiological data, either a line list, to provide a row-by-row account of cases in an infectious disease outbreak, or a contacts table, to provide a history of which individuals were in contact with which others.
+
+The main function in the {simulist} package is `sim_linelist()`. This functions takes in arguments that control the dynamics of the outbreak, such as the serial interval, and outputs a line list table (`<data.frame>`) with case information for each infected individual.
+
+For this introduction we will simulate a line list for a COVID-19 (SARS-CoV-2) outbreak. This will require two R packages: {simulist}, to produce the line list, and {epiparameter} to provide epidemiological parameters, such as onset-to-death delays.
```

Comment:
```suggestion
For this introduction we will simulate a line list for the early stages of a COVID-19 (SARS-CoV-2) outbreak. This will require two R packages: {simulist}, to produce the line list, and {epiparameter} to provide epidemiological parameters, such as onset-to-death delays.
```

Reply:
Updated in 91286578306f249a66ce386bdc50d7ac1e71f280.